README.md

```diff
@@ -1,60 +1,123 @@
+- testing with [minitest][minitest],
+- submitting the test results to [Coveralls][Coveralls]
+to obtain a test coverage report,
+- running [Codeclimate][CodeClimate] to get a quality control report.
+
+### [minitest][minitest]
+Our test suite uses [minitest][minitest] and extensions.
+See our [Gemfile][Gemfile] for more details.
+The test suite is run as part of our CI, and also can be run locally.
+
+### [Coveralls][Coveralls]
+We use [Coveralls][Coveralls] to obtain a
+[report about test coverage][coveralls_build].
+Coveralls is run as part of our CI, and also can be run locally.
+Its reports include total coverage and coverage for each line of code.
+The reports highlight newly uncovered lines
+so that we can maintain a high degree of coverage.
+
+### [Codeclimate][CodeClimate]
+We use [Codeclimate][CodeClimate]
+and plugins, like [RuboCop][RuboCop] and [Brakeman][Brakeman],
+to help maintain and improve code quality, including a consistent style.
+See our [Codeclimate configuration file](/.codeclimate.yml) for more details.
 
-Source code control: The source code for MO is currently managed using git
-and is hosted on github. The URL for the MO source repository is
-<https://github.com/MushroomObserver/mushroom-observer>.
+### [RuboCop][RuboCop]
+We use [RuboCop][RuboCop] and extensions to help with
+code quality and consistentcy of style.
+See our [RuboCop configuration file](/.rubocop.yml) for more details.
+[RuboCop][RuboCop] is run as part of our CI, and also can be run locally.
+(We are a long way from where we want to be.
+We have hundreds of offenses to fix.
+See our [Ruboco Todo file](.rubocop_todo.yml).)
 
+### [Brakeman][Brakeman]
+We use [Brakeman][Brakeman] to highight some security issues.
+
+----
+
+## DEVELOPMENT
 To get started developing we recommend that you head over to
 <https://github.com/MushroomObserver/developer-startup> which provides
-instructions for getting a development environment setup on Mac, Linux
-and Windows machines.
+instructions for getting a virtual machine development environment set up on
+Mac, Linux and Windows machines.
 
-Other READMEs
--------------
+----
+
+## Other READMEs
 
 The following files should provide useful information for working with the
-Mushroom Observer code base.  Note that we are not great at keeping these up
-to date, so be sure to take them with a grain of salt proportional to the
-Last Revised date.
+Mushroom Observer code base. These files are quite out-of-date,
+so be sure to take them with a grain of salt.
 
-README_CODE: Provides a snapshot of the major components of Mushroom Observer.
+[README_CODE](README_CODE):
+Provides a snapshot of the major components of Mushroom Observer.
 
-README_DEVELOPMENT_INSTALL: Describes the steps needed for a developer to get
-the code running locally. Though we recommend following the process outlined
-at <https://github.com/MushroomObserver/developer-startup>.
+[README_DEVELOPMENT_INSTALL](README_DEVELOPMENT_INSTALL):
```

Comment:
Should I remove README_DEVELOPMENT_INSTALL and this reference to it? The file is over 10 years old.